src/QsCompiler/DataStructures/Serialization.fs

```diff
@@ -15,18 +15,6 @@ open Newtonsoft.Json
 open Newtonsoft.Json.Linq
 open Newtonsoft.Json.Serialization
 
-
-type NonNullableConverter<'T when 'T: equality and 'T: null>()  =
```

Comment:
Did you double check whether the extension including this change works fine when opening an older project? I.e. the references serialized based on the old data structures are properly loaded?

Reply:
I opened a project using version 0.12.20100504 of the SDK with a language server built from this branch, and it worked.

Reply:
Thanks for confirming!